doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,329 @@
+	
+    Access-Control-Expose-Headers: Fiware-Correlator, Fiware-Total-Count, Location.
+
+[Top](#top)
+
+
+## 16. Allow
+
+Allow HTTP header is used in response notification.
+It notify that which methods are allowed by the specified host/resource.
+
+    Allow: GET
+
+[Top](#top)
+		
+    
+## 17. Location
+
+Location HTTP header is used in responses.
+Response contains a Location header which holds the subscription ID: a 24 digit hexadecimal number used for updating and cancelling the subscription.
+we use `-v` in requests to get the Location header in the response.
+	
+    curl -v localhost:1026/v2/subscriptions -s -S -H 'Content-Type: application/json' -d @- <<EOF
+    {
+     //payload
+    }
+    EOF
+	
+And the response that we will get:
+	
+    Location: /v2/subscriptions/57458eb60962ef754e7c0998
+	
+[Top](#top)
+
+
+## 18. Ngsiv2-AttrsFormat
+
+It is a response HTTP header in Orion.
+Notifications must include the `Ngsiv2-AttrsFormat` HTTP header with the value of the format of the associated subscription, 
+so that notification receivers are aware of the format without needing to process the notification payload.
+Note that if a custom payload is used for the notification then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
+in the notification.	
+Ngsiv2-AttrsFormat is a NON-MODIFIABLE header in custom notifications.
+Any attempt of doing so (e.g. `"httpCustom": { ... "headers": {"Ngsiv2-Attrsformat": "something"} ...}` will be ignored.
+	
+    Ngsiv2-Attrsformat: normalized
+
+[Top](#top)	
+
+
+## 19. Fiware-Service
+
+Fiware-Service is an optional header used in requests.
+When `-multiservice` is used Orion includes the `Fiware-Service` header in the notifyContextRequest and notifyContextAvailability request messages associated to subscriptions in the given tenant/service (except for the default service/tenant, in which case the header is not present).
+		
+    POST http://127.0.0.1:9977/notify    
+    Content-Length: 725    
+    User-Agent: orion/0.13.0    
+    Host: 127.0.0.1:9977      
+    Accept: application/json    
+    Fiware-Service: t_02    
+    Content-Type: application/json    
+    {    
+	   ...    
+    } 
+      
+ Regarding service/tenant name syntax, it must be a string of alphanumeric characters (and the `_` symbol). 
+ Maximum length is 50 characters, which should be enough for most use cases. 
+ Orion Context Broker interprets the tenant name in lowercase, thus, although you can use tenants such as in updateContext `MyService` 
+ it is not advisable, as the notifications related with that tenant will be sent with `myservice` and, 	in that sense, 
+ it is not coherent the tenant you used in updateContext compared with the one that Orion sends in notifyContextRequest.
+	
+[Top](#top)
+
+
+## 20. Fiware-Servicepath
+
+Fiware-ServicePath is an optional header used in requests.
```

Comment:
It is used in any kind of HTTP transaction (incoming/outcoming requests and outcoming responses) managed in Orion.

Reply:
In addition, I'd say that L259 is redundant and should be removed.

Reply:
Fixed in a427bd9